images/hub/jupyterhub_config.py

```diff
@@ -139,7 +140,8 @@
     org_whitelist = get_config('auth.github.org_whitelist', [])
     if len(org_whitelist) != 0:
         c.GitHubOAuthenticator.github_organization_whitelist = org_whitelist
-        c.GitHubOAuthenticator.scope = ['read:org'] # required for private membership
+        if not auth_scopes:
+            c.Authenticator.scope = ['read:org'] # required for private membership
```

Comment:
This feel like the wrong place to embed the logic for default scopes, but I don't have a better suggestion.